packages/observability/__tests__/logger-functions.test.ts

```
@@ -150,4 +150,232 @@ describe('logger Functions', () => {
   });
+
+  describe('server environment behavior', () => {
+    let originalWindow: typeof window;
+    let originalGlobal: typeof globalThis;
+
+    beforeEach(() => {
+      originalWindow = (globalThis as any).window;
+      originalGlobal = globalThis;
+      
+      // Mock server environment
+      delete (globalThis as any).window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)